src/routes/Onboarding/ImportAccount.tsx

```diff
@@ -47,7 +49,15 @@ const EncodedKeyTab: FC<DesktopModeProps> = ({ onImport }) => {
           variant="primary"
           isDisabled={!data}
           onClick={() => {
```

Comment:
Can you rename this to importByData like the hook actually has? This is a misnomer, the data is an encoded account, not spending key. Code should work as is, but it is misleading.

Reply:
Done